app/assets/stylesheets/dlme.scss

```diff
@@ -275,6 +281,11 @@ body {
       text-decoration: underline;
     }
   }
+
+  // hide the "sign in" link
+  &[href="/users/sign_in"] {
```

Comment:
Can we remove this and just not render the link https://github.com/sul-dlss/dlme/blob/main/app/views/shared/_user_util_links.html.erb#L41-L45

Reply:
Yeah, good call. made that change